schema/training-module.js

```diff
@@ -1,9 +1,42 @@
 const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
 
 class TrainingModule extends BaseModel {
   static get tableName() {
     return 'trainingModules';
   }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [],
+      additionalProperties: false,
+      properties: {
+        id: { type: 'string', pattern: uuid.v4 },
+        'migrated_id': { type: ['string', 'null'] },
+        module: { type: 'string' },
+        species: {
+          type: 'array',
+          items: {
+            type: 'string'
+          }
+        },
+        pass_date: { type: 'string', format: 'date-time' },
+        not_applicable: { type: 'boolean' },
+        accrediting_body: { type: 'string' },
+        other_accrediting_body: { type: 'string' },
+        certificate_number: { type: 'string' },
+        exemption: { type: 'boolean' },
+        exemption_description: { type: 'text' },
```

Comment:
should this be type: `string` rather than `text`?

I think json schema only has string https://json-schema.org/understanding-json-schema/reference/type.html

Reply:
Created https://jira.digital.homeoffice.gov.uk/browse/AS-663.
Will be merging this branch.